mani_skill/utils/building/articulation_builder.py

```diff
@@ -56,7 +60,9 @@ def create_link_builder(self, parent: LinkBuilder = None):
 
         return builder
 
-    def build_entities(self, fix_root_link=None, name_prefix=""):
+    def build_entities(
```

Comment:
make this a private function (_build_entities) actually since none of the code here should be used by users. It is just for creating SAPIEN objects and is used only by. def build

Also hide the original `def build_entities` (just raise an error if it is called explicitly) or delete it if possible

Reply:
made private function and throw error if build_entities called directly